ethereum_history_api/tests/src/get_transaction.e2e.test.ts

```diff
@@ -0,0 +1,38 @@
+import { beforeEach, describe, expect, it } from 'vitest';
+import { updateNestedField, incHexStr } from 'noir-ethereum-api-oracles';
+import { Abi, InputMap, abiEncode } from '@noir-lang/noirc_abi';
+
+import getTransactionVerifier from '../../contracts/out/GetTransactionUltraPLONKVerifier.sol/UltraVerifier.json';
+import getTransaction from '../../../target/get_transaction.json';
+
+import { readProofData } from './proofDataReader.js';
+import { FoundryArtefact, deploySolidityProofVerifier } from './solidityVerifier.js';
+
+export const abi = getTransaction.abi as unknown as Abi;
+
+describe('get_transaction', async () => {
+  let proof: Uint8Array;
+  let inputMap: InputMap;
+  const proofVerifier = await deploySolidityProofVerifier(getTransactionVerifier as FoundryArtefact);
+
+  beforeEach(async () => {
+    ({ proof, inputMap } = await readProofData('get_transaction'));
+  });
+
+  it('proof verification successes', async () => {
```

Comment:
```suggestion
  it('proof verification succeeds', async () => {
```
Let's change it in all e2e tests?